src/main/java/org/sonatype/nexus/ci/nvs/NvsMessageAction.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.sonatype.nexus.ci.nvs
+
+import javax.annotation.CheckForNull
+
+import hudson.model.Action
+
```

Comment:
Would be nice if we add a comment here about why this implementation returns null.

Reply:
Good idea - I updated and copied the comment from `PolicyEvaluationProjectAction`